anymail/backends/mailpace.py

```diff
@@ -0,0 +1,190 @@
+                                    )
+                        else:
+                            continue  # No errors found in this field; continue to next field
+            else:
+                raise AnymailRequestsAPIError(
+                    email_message=message,
+                    payload=payload,
+                    response=response,
+                    backend=self,
+                )
+
+        return dict(recipient_status)
+
+
+class MailPacePayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+        }
+        self.server_token = backend.server_token  # esp_extra can override
+        self.to_cc_and_bcc_emails = []
+        self.merge_data = None
+        self.merge_metadata = None
+        super().__init__(message, defaults, backend, headers=headers, *args, **kwargs)
+
+    def get_request_params(self, api_url):
+        params = super().get_request_params(api_url)
+        params["headers"]["MailPace-Server-Token"] = self.server_token
+        return params
+
+    def serialize_data(self):
+        return self.serialize_json(self.data)
+
+    #
+    # Payload construction
+    #
+
+    def init_payload(self):
+        self.data = {}  # becomes json
+
+    def set_from_email(self, email):
+        self.data["from"] = email.address
+
+    def set_recipients(self, recipient_type, emails):
+        assert recipient_type in ["to", "cc", "bcc"]
+        if emails:
+            # Creates to, cc, and bcc in the payload
+            self.data[recipient_type] = ", ".join([email.address for email in emails])
+            self.to_cc_and_bcc_emails += emails
+
+    def set_subject(self, subject):
+        self.data["subject"] = subject
+
+    def set_reply_to(self, emails):
+        if emails:
+            reply_to = ", ".join([email.address for email in emails])
+            self.data["replyto"] = reply_to
+
+    def set_text_body(self, body):
+        self.data["textbody"] = body
+
+    def set_html_body(self, body):
+        self.data["htmlbody"] = body
+
+    def make_attachment(self, attachment):
+        """Returns MailPace attachment dict for attachment"""
+        att = {
+            "name": attachment.name or "",
+            "content": attachment.b64content,
+            "content_type": attachment.mimetype,
+        }
+        if attachment.inline:
+            att["cid"] = "cid:%s" % attachment.cid
+        return att
+
+    def set_attachments(self, attachments):
+        if attachments:
+            self.data["attachments"] = [
+                self.make_attachment(attachment) for attachment in attachments
+            ]
+
+    def set_tags(self, tags):
+        if tags:
+            if len(tags) == 1:
+                self.data["tags"] = tags[0]
+            else:
+                self.data["tags"] = tags
```

Comment:
Please also implement `set_esp_extra`. This helps future-proof Anymail when MailPace adds new options to the send API.

(Also, you have code above that supports overriding the server token on individual messages, via esp_extra, so that needs to be handled here. I'm guessing this is important for domain-specific API keys. You can copy from the [Postmark backend](https://github.com/anymail/django-anymail/blob/v10.2/anymail/backends/postmark.py#L386-L389), which has exactly the same feature.)